PharmacyManagementAPI/Controllers/PharmacyController.cs

```diff
@@ -18,50 +21,81 @@ public PharmacyController(IPharmacyService service)
         [HttpGet]
         public async Task<ActionResult> GetAllPharmacies()
         {
-            var pharmacies = await _service.GetAllPharmaciesAsync();
+            try
```

Comment:
Great job on error handling with try catch. Explore other ways to handle errors - 2 other ways are Exception Filters and Exception Middleware